ui.apps/src/main/content/jcr_root/apps/acs-commons/components/utilities/report-builder/columns/property.jsp

```diff
@@ -0,0 +1,31 @@
+<%--
+  #%L
+  ACS AEM Commons Package
+  %%
+  Copyright (C) 2017 Adobe
```

Comment:
2022

Reply:
Also, i was surprised this is only used in the `.../columns/data/results.jsp` that sound right?

I originally broke it out anticipating it being used in several results.jsp